arcgis-ios-sdk-samples/Layers/Blend renderer/README.md

```diff
@@ -1,19 +1,37 @@
 # Blend renderer
 
-This sample demonstrates how to use blend renderer on a raster layer. You can get a hillshade blended with either a colored raster or color ramp.
+Blend a hillshade with a raster by specifying the elevation data. The resulting raster looks similar to the original raster, but with some terrain shading, giving it a textured look.
 
-## How to use the sample
+![Blend renderer default renderer](blend-renderer-1.png)
+![Edited renderer 1](blend-renderer-2.png)
+![Edited renderer 2](blend-renderer-3.png)
 
-Tap on the `Edit renderer` button in the toolbar to change the settings for the blend renderer. The sample allows you to change the Altitude, Azimuth, Slope type and Color ramp type. You can tap on the Render button to update the raster. If you use `None` as the color ramp type, colored raster is blended with the hillshade output. For all the other types a color ramp is used.
 
-![](image1.png)
-![](image2.png)
-![](image3.png)
+## How to use the sample
 
+Tap the bottom button to choose and adjust the altitude, azimuth, slope type and color ramp type settings to update the image.
 
 ## How it works
 
-The sample uses `AGSBlendRenderer` class to generate blend renderers. The settings provided by the user are put in the initializer to get a new renderer and the renderer is then set on the raster.
+1. Create an `AGSRaster` object from a raster file.
+2. Create an `AGSRasterLayer` object from the raster.
+3. Create an `AGSBasemap` object from the raster layer and set it to the map.
+4. Create another `AGSRaster` object for elevation from a grayscale raster file.
+5. Create an `AGSBlendRenderer` object, specifying the elevation raster, color ramp, and other properties.
+    - If you specify a non-null color ramp, use the elevation raster as the base raster in addition to the elevation raster parameter. That way, the color ramp is used instead of the satellite imagery.
```

Comment:
```suggestion
    * If you specify a non-null color ramp, use the elevation raster as the base raster in addition to the elevation raster parameter. That way, the color ramp is used instead of the satellite imagery.
```